cardano-api/src/Cardano/Api/Script.hs

```diff
@@ -113,6 +113,7 @@ module Cardano.Api.Script (
 import qualified Data.ByteString.Lazy as LBS
 import           Data.ByteString.Short (ShortByteString)
 import qualified Data.ByteString.Short as SBS
+import           Data.Default.Class
```

Comment:
I don't understand what the benefit is of using `Data.Default.Class` over defining a single default value.